app/doc/DocVersions.scala

```diff
@@ -1,5 +0,0 @@
-package doc
-
-object DocVersions {
-  val Latest = "1.0.x"
```

Comment:
The problem with the previous approach was that it was tied to ConductR. Project-doc should remain agnostic with exception to the module declarations and how they relate to URI paths. 

> On 18 Sep 2015, at 7:17 am, Markus Jura <notifications@github.com> wrote:
> 
> In app/doc/DocVersions.scala:
> 
> > @@ -1,5 +0,0 @@
> > -package doc
> > -
> > -object DocVersions {
> > -  val Latest = "1.0.x"
> Why do we not encapsulate the versions in an singleton object as before. Currently we are using the versions in multiple places:
> 
> https://github.com/huntc/project-doc/blob/more-versions/app/modules/ConductRDocRendererModule.scala#L36
> https://github.com/huntc/project-doc/blob/more-versions/app/controllers/Application.scala#L90
> Therefor we should encapsulate the strings to constants inside of a singleton object.
> 
> —
> Reply to this email directly or view it on GitHub.
>